### PR TITLE
Test service providers

### PR DESCRIPTION
**Pull request by adamtobey**

### CODE REVIEW

**Comment by benjaminclauss** [comment]
The provider route isn't the same as the demo.

https://github.com/jannunzi/cs4500-sp19s2-client-react/blob/master/src/App.js

File: src/App.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ class App extends Component {
                   component={Home}/>
 
               <Route
```
Comment (benjaminclauss) [inline src/App.js]: Create a new individual provider route.

Comment (benjaminclauss) [reply src/App.js]: https://github.com/jannunzi/cs4500-sp19s2-client-react/blob/master/src/App.js

Comment (adamtobey) [reply src/App.js]: This is actually the link to the provider search for service with id 123. We're hard coding 123 right now until we have the full search flow.

**Pull request merged**